FueledUtils/LabelWithTitleAdjustment.swift

```diff
@@ -36,7 +46,7 @@ open class LabelWithTitleAdjustment: UILabel {
 
 	private func setAdjustedAttributedText(_ text: NSAttributedString?) {
 		guard let text = text else {
-			self.attributedText = nil
+			super.attributedText = nil
```

Comment:
I'd add a comment that this is done in order to avoid recursion. It's not immediately obvious.

Reply:
👍